_posts/2021-01-27-2021-kick-off.md

```diff
@@ -0,0 +1,54 @@
+---
+
+title: "2021 kick-off"
+author: [molnarm84, huszerlbence]
+image: "../img/posts/2021-kickoff.jpg"
+tags: [OKR, SNaaS, saas, infra, planning, q1]
+redirect_to: https://www.sensenet.com/blog/2021-01-27-2021-kick-off
+
+---
+
+As the holiday season ended, we have so much to do at the beginning of the year to launch SNaaS according to our plan. Get a short insight what we have in mind for Q1.
+
+---
+
+At the end of 2020 we sat down to clarify our goals for the next year (both on company and team level) and defined the way to reach them. We came up with the idea to use OKR for that so we started to familiarize with the methodology and defined objectives and key results for Q1.
+
+# OKR in a tiny nutshell
+
+"Objectives and Key Results (OKR) is a critical thinking framework and goal setting methodology that helps companies to align goals and ensure everyone is working collaboratively on goals that really matter." (source: [weekdone.com](https://weekdone.com))
+
+We found that Weekdone offers everything we need at the moment to start our journey so we decided to give it a go. Since every department has a role in reaching the goals, we applied the methodology to the whole company.
```

Comment:
```suggestion
We found that [Weekdone](https://weekdone.com/) offers everything we need at the moment to start our journey so we decided to give it a go. Since every department has a role in reaching the goals, we applied the methodology to the whole company.
```